dotcom-rendering/index.d.ts

```diff
@@ -5,7 +5,6 @@
 // Pillars are used for styling
 // RealPillars have pillar palette colours
 // FakePillars allow us to make modifications to style based on rules outside of the pillar of an article
-// These are partialy kept for Google Analytics purposes
```

Comment:
If this is true, should they be removed? Are they used for anything else?

Reply:
I might misunderstood that. Isn't partially kept means it is kept for different reasons as well and one of them is Google Analytics? 

What should be removed? Is it the following three types? 

I checked now and the `LegacyPillar` is used in the codebase.